demos/toolbar/menu-toolbar.html

```diff
@@ -52,17 +52,17 @@
         <section class="mdc-toolbar__section mdc-toolbar__section--align-end" role="toolbar">
           <a href="#" class="material-icons mdc-toolbar__icon" aria-label="Download" alt="Download">file_download</a>
           <a href="#" class="material-icons mdc-toolbar__icon" aria-label="Print this page" alt="Print this page">print</a>
-          <div class="mdc-menu-anchor">
           <a href="#" class="material-icons mdc-toolbar__icon toggle" aria-label="Bookmark this page" alt="Bookmark this page">more_vert</a>
```

Comment:
Incidentally, there's some copypasted label/alt attribute values from that original commit here too.

Reply:
Sorry, I am not sure if I followed this comment...

Reply:
the label and alt on the overflow button say "Bookmark this page" when they should probably say "More".

Reply:
Nice catch. It seems one of earlier commit updated the icon but not aria tag.

I am thinking changing this back to `bookmark` instead of `more_vert` because `more_vert` seems a pattern that user expect menu will pop out?